site/en/docs/bazel-and-cpp.md

```diff
@@ -82,3 +82,22 @@ Follow these guidelines for include paths:
    use the [`include_prefix`](/reference/be/c-cpp#cc_library.include_prefix) and
    [`strip_include_prefix`](/reference/be/c-cpp#cc_library.strip_include_prefix)
    arguments on the `cc_library` rule target.
+
+### Toolchain features {:#toolchain-features}
+
+The following optional [features](/docs/cc-toolchain-config-reference#features)
+can improve the hygiene of a C++ project. They can be enabled via the
+`--features` command-line flag or via the `features` attribute of
```

Comment:
Linter requests to replace "via" with "using".
Linter points to https://developers.google.com/style/word-list#via
I admit that I don't know why, but I am inclined to believe the general reason the word list gives for "don't use" words: "The term might be particularly ambiguous or it might have an offensive or non-inclusive association."